README.md

```diff
@@ -55,6 +55,7 @@ Select *Stash Pull Request Builder* then configure:
 - Rebuild if destination branch changes:
 - Build only if Stash reports no conflicts: this should be set if using the merge branch to avoid issues with out of data merge branch in stash
 - Build only if PR is mergeable (note this will stop the PR being built if you have required approvers limit set >0 and the PR hasn't been approved)
+- Probe the Stash REST API for merge status just so it recalculates the refspec: This probes the Stash REST API endpoint that causes recalculation of Git refspecs (see [JENKINS-35219](https://issues.jenkins-ci.org/browse/JENKINS-35219) and [Atlassian KB 239988](https://answers.atlassian.com/questions/239988/change-pull-request-refs-after-commit-instead-of-after-approval-or-workaround) for details). Has no impact on proceeding with the build itself, unlike the "Build only if..." options above.
```

Comment:
minor: I think this can be shortened, the first two sentences kind of say the same, why not just `Probe the Stash REST API for merge status just so it recalculates the refspec: see [JENKINS-35219](https://issues.jenkins-ci.org/browse/JENKINS-35219) ... `

Reply:
Good catch. The part before the colon is the (later shortened) option name as seen in configs. The rest is layman's description (a revision of which is now also the help text for the option).